SPECS/expat.spec

```diff
@@ -60,334 +62,30 @@ make check
 %doc AUTHORS Changes
 %license COPYING
 %{_bindir}/*
-%{_libdir}/lib*.so.*
-%{_mandir}/*/*
+%{_libdir}/libexpat.so.1
+%{_libdir}/libexpat.so.1.*
+%if 0%{?xenserver} < 9
+%{_docdir}/%{name}/AUTHORS
+%{_docdir}/%{name}/Changes
```

Comment:
I don't see the point :thinking: 

Do you have an idea why XS has done this (it's not in the EL spec file)?

There's `%doc` already to include these two files, in theory.